rpm/filescanner.go

```diff
@@ -0,0 +1,89 @@
+package rpm
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"runtime/trace"
+
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/indexer"
+)
+
+const (
+	scannerName    = "rpm"
+	scannerVersion = "1"
```

Comment:
would updating this (or even just introducing it) trigger reindexing?

Reply:
It should cause the index_state to change, which an external application can use to trigger re-indexing